yt/data_objects/tests/test_firefly.py

```diff
@@ -3,16 +3,38 @@
 from yt.testing import fake_particle_ds, requires_module
 
 
-@requires_module("firefly_api")
-def test_firefly_JSON_object():
+@requires_module("Firefly")
+def test_firefly_JSON_string():
+
+    ds = fake_particle_ds()
+    ad = ds.all_data()
+    reader = ad.create_firefly_object(
+        None,
+        velocity_units="cm/s",
+        coordinate_units="cm",
+    )
+
+    reader.dumpToJSON(write_jsons_to_disk=False)
+
+    if len(reader.JSON) == 0:
+        raise ValueError(
+            "reader.JSON was not output to string correctly."
+            + " either Firefly is damaged or needs a hotfix-- try reinstalling."
+            + " if that doesn't work contact the developers"
+            + " at github.com/ageller/Firefly/issues."
+        )
```

Comment:
I think this should be split into an assert statement and a comment

Reply:
okay! Made the change :)